### PR TITLE
Added PMTiles #17

### LINKED ISSUE #17: Add PMTiles
- href: URL to the .pmtiles "file"
- type: What's a good media type?
- rel: pmtiles
- What other metadata might be useful to specify in the link for visualization purposes?
  - In the OL example `tileSize` is set
  - MapLibre has quite a number of options set: `type` and `encoding` in the source, `source-layer`, `type` and `paint` in the layer

### PR DESCRIPTION
**Pull request by m-mohr**
OL STAC implementation: https://github.com/m-mohr/ol-stac/pull/4

### CODE REVIEW

**Comment by m-mohr** [comment]
Any thoughts @bdon?

File: json-schema/schema.json [hunk 1]
```diff
@@ -140,6 +141,27 @@
...
+                }
+              }
+            },
+            "then": {
+              "properties": {
+                "pmtiles:layers": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "minLength": 1
```
Comment (bdon) [inline json-schema/schema.json:160]: should this be 0 in the case it's a raster tileset?

Comment (m-mohr) [reply json-schema/schema.json:160]: No, in this case you just omit the array or provide an empty array, but not an array with an empty string.

Comment (bdon) [reply json-schema/schema.json:160]: From reading [https://github.com/mapbox/vector-tile-spec/tree/master/2.1](the MVT spec) I think empty string is a valid layer name :(

Comment (m-mohr) [reply json-schema/schema.json:160]: Looks like it, indeed, but I want to see the person who complains about unsupported empty layer names ;-)

**Pull request merged**